CacaraGyorgis/CCRG001.xml

```diff
@@ -964,14 +964,14 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
               </decoNote>
 
               <decoNote type="miniature" xml:id="d2">
-                <locus target="#42v"/>
+                <locus target="#54r"/>
                 <desc>A miniature of the art theme <ref type="authFile"
                     corresp="AT1036MercyCovenant"/> with one episode where Our Lord <persName
                     ref="PRS5684JesusCh"/> and His mother Saint <persName ref="PRS6819Mary"/>
                   sitting in the center of the scene and surrounded by angels.</desc>
               </decoNote>
               <decoNote type="miniature" xml:id="d3">
-                <locus target="#10r"/>
+                <locus target="#67vb-68rb"/>
```

Comment:
for a range of folio numbers, use the attributes to and from:
` <locus from="67vb" to="68rb"/>`